### PR TITLE
Classification And Validation Framework Test Cases

### PR DESCRIPTION
**Pull request by ravipwaghmare**
Added Test Cases for Validation and Classification Framework
@maelk @digambar15 Kindly review.

### CODE REVIEW

File: hcmanager/hcmanager_test_data.go [hunk 1]
```diff
@@ -60,12 +60,16 @@ func getNamespace() string {
 	return "metal3"
 }
 
+func getTestProfileName() string {
+	return "testProfileName"
+}
+
```
Comment (maelk) [inline hcmanager/hcmanager_test_data.go:66]: Something I did not realize at first, the naming of this file is incorrect. If you are using it for test only, it should end in _test.go . so you need to change it to hcmanager_data_test.go

Comment (ravipwaghmare) [reply hcmanager/hcmanager_test_data.go:66]: @maelk Done

**Review by digambar15 — APPROVED** [review]
Good Job! Thanks!

**Pull request merged**